android/app/src/main/java/com/audiobookshelf/app/player/PlayerNotificationService.kt

```diff
@@ -258,7 +258,7 @@ class PlayerNotificationService : MediaBrowserServiceCompat()  {
     playerNotificationManager.setPriority(NotificationCompat.PRIORITY_MAX)
     playerNotificationManager.setUseFastForwardActionInCompactView(true)
     playerNotificationManager.setUseRewindActionInCompactView(true)
-    playerNotificationManager.setSmallIcon(R.drawable.exo_icon_localaudio)
+    playerNotificationManager.setSmallIcon(R.drawable.icon_monochrome)
```

Comment:
Also updated the icon in the notification player since I pulled the abs font icon out to a normal icon for the widget